server/server.js

```diff
@@ -20,9 +16,9 @@ mongoose.connect('mongodb://localhost:27017/eventplannerDB', {
   useCreateIndex: true
 });
 
-app.use(bodyParser.json());
+//app.use(bodyParser.json());
```

Comment:
Took me forever to figure out why the backend wasn't receiving the body of the request. Front end needs the app.use(bodyParser.json()) otherwise the request fails.